code/scripting/api/libs/base.cpp

```diff
@@ -75,6 +75,25 @@ ADE_FUNC(createOrientation,
 	return ade_set_error(L, "o", l_Matrix.Set(matrix_h()));
 }
 
+ADE_FUNC(createOrientationFromVectors, l_Base, "[vector fvec, vector uvec, vector rvec]",
+	"Given 0 to 3 arguments, creates an orientation object from vectors.",
```

Comment:
It's probably a good idea to document that passing no parameters will result in the identity matrix. Not strictly necessary but could reduce confusion for people who don't read the code.